### ISSUE TITLE
The shortcut keys are available only for screen reader dependent users and action performed using shortcut keys is not clear

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5219]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Screen reader dependent users will be confused if same action can be performed by tab, shift tab and with shortcut keys.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator, Windows: Chrome with NVDA, Windows: Chrome/Firefox with JAWS

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true

### How to reproduce the issue?

1. Open [https://www.microsoft.com/en-us/surface?at_preview_token=4FhxMZZq5RXQNNpOOuFqCLBpWwq30eLAUx7_sjX2n68&at_preview_index=1_2&at_preview_listed_activities_only=true](https://wwwppe.microsoft.com/en-us/surface) in Edge. 
2. Start NVDA. 
3. Turn off your monitor. 
4. Tab to the ‘Help me choose’ button and open the window. Focus will be on the close “X”. 
5. Tab (x2). 
6. Use the down arrow key to access the message content. 
7. Navigate the prompts and selection until you get to the button selections. 
8. Do you hear the shortcut key narrated? 
9. Select [Alt]+[Shift]+[A]  
10. What happened? Is it clear what the shortcut does?

### What do you expect?

Clear instructions should be given to user on what action will be performed by using keyboard shortcut keys ‘[Alt]+[Shift]+[A]’
Observation: Here keyboard shortcut keys ‘[Alt]+[Shift]+[A]’ move the user from one button to the next, however this action/purpose is not conveyed to the user. Is the shortcut key intended to move the user from one button to the next? Why are the shortcut keys needed when the user can [Tab]/[Shift]+[Tab] or [up/down arrow] keys? 

### What actually happened?

The access shortcut keys ‘[Alt]+[Shift]+[A]’ that the user is instructed to use for the selection buttons just move the user from one button to the next, however this action/purpose is not conveyed to the user.

### Do you have any screenshots or recordings to repro the issue?

<img width="458" alt="image" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/df2950e7-2e97-48c9-9dbd-76bf11de1626">

### MAS reference

https://aka.ms/MAS1.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev2;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by OEvgeny** [comment]
I can't reproduce the issue in the upstream anymore. There is no suggested actions

**Comment by Kirank915** [comment]
Verified the issue on https://microsoft.github.io/BotFramework-WebChat/01.getting-started/a.full-bundle/ under Windows 11 Version 24H2 (OS build: 26063.1), Edge Version 119.0.2151.44 (official build) dev (64-bit) and observed that now there is no suggested shortcut suggested announcement from screen readers

Hence issue is fixed, closed the bug
